### ISSUE TITLE
Feed Input: Accessibility Issues

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#4664]
Feed Input Accessibility Issues

Attached the screenshot:

[Feed input .docx](https://github.com/SAP/fundamental-ngx/files/5980054/Feed.input.docx)

**Comment by Karthickc1994** [comment]
Hello Colleague,

Retest has been taken results are below:

Issue1: Fixed.

Issue2: 
observed behavior: 
The in-place label text 'Add comments' does not meet the required ratio of 7:1.

Expected behavior: 
The in-place label text 'Add Comments'  should  satisfy the required ratio of 7:1.
![image](https://user-images.githubusercontent.com/105037808/170972338-edce8052-30d3-4d7c-b89a-c8e8d3275ccd.png)

 

Issue3: Fixed.

Retest Environment:
URL: https://fundamental-ngx.netlify.app/#/platform/feed-input
Browser:  Chrome Version 102.0.5005.63 (Official Build) (64-bit)
Screen reader; JAWS 2022.2204.20 ILM 

Regards,
Karthikeyan.c

Regards,
Karthikeyan.c
